src/main/resources/db/migration/V003__order.sql

```diff
@@ -0,0 +1,43 @@
+CREATE TABLE IF NOT EXISTS orders (
+  id  BIGINT AUTO_INCREMENT NOT NULL,
+  uuid  VARCHAR(36) NOT NULL UNIQUE,
+  user_id  BIGINT NOT NULL,
+  agent_id  BIGINT NOT NULL,
+  created_on  TIMESTAMP NOT NULL,
+  last_modified TIMESTAMP,
+  SKU VARCHAR(255),
+  order_type VARCHAR(100),
+  order_status VARCHAR(100),
+  currency VARCHAR(36) NOT NULL,
+  total_price DECIMAL(10,2),
+  shipping_instruction TEXT,
+  order_note  TEXT,
+  PRIMARY KEY (id)
+)
+  ENGINE=InnoDB
+  DEFAULT CHARSET=utf8;
+
+CREATE TABLE IF NOT EXISTS order_2_item (
```

Comment:
add `order_id` to items instead. This prob make more sense for a many-to-many relationship. Order an item is one-to-many